test/test_circuitzoo.jl

```diff
@@ -0,0 +1,65 @@
+using Test
+using QuantumSavory
+using QuantumSavory.CircuitZoo: EntanglementSwap, Purify2to1, Purify3to1, PurifyStringent, PurifyExpedient
+
+
+const T2=0
+const perfect_pair = (Z1⊗Z1 + Z2⊗Z2) / sqrt(2)
+const perfect_pair_dm = SProjector(perfect_pair)
+const mixed_dm = MixedState(perfect_pair_dm)
+noisy_pair_func(F) = F*perfect_pair_dm + (1-F)*mixed_dm
+
+function entangled_registers(size, purity, T2=0)
+    regA = Register([Qubit() for  _ in 1:size],
+                    [QuantumOpticsRepr() for  _ in 1:size], 
+                    [T2Dephasing(T2) for  _ in 1:size])
```

Comment:
it will probably be easier for the tests, if you do not add noise backgrounds. Actually, you can just write `Register(size)` to create a register of `size` qubits with QuantumOptics representation and no noise.